src/components/MultiLink/Legend.vue

```diff
@@ -58,6 +59,45 @@ export default Vue.extend({
     nodeColorScale() {
       return store.getters.nodeColorScale;
     },
+
+    columnTypes() {
+      const typeMapping: { [key: string]: string} = {};
```

Comment:
There's a slight formatting issue on this line. Do you have prettier or eslint or something else running on this repo?

Reply:
We're using eslint, but it doesn't complain about this line. Either way, I've fixed it in 0e719ac